BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee

```diff
@@ -5,7 +5,7 @@ FullPageFormView = OrchestraView.extend(
     @options = options
     @options.listUrl = appRouter.generateUrl('listEntities', entityType: options.entityType) if options.listUrl == undefined
     @completeOptions(@options.element) if @options.element != undefined
-    @events = {}
+    @events = @events || {}
```

Comment:
Why do you exttend the event array?

If you do not declare any array in the initialize, you do not need to initialize the array